pkg/models/v1/models.go

```diff
@@ -222,6 +222,28 @@ func GetVASPEmailLog(vasp *pb.VASP) (emails []*EmailLogEntry, err error) {
 	return emails, nil
 }
 
+// Create and add a new entry to the EmailLog on the extra data on the Contact record.
+func (x *Contact) AppendEmailLog(reason, subject string) {
```

Comment:
Awesome! If I could make one cosmetic suggestion, use the `c` in the pointer receiver instead of `x`. Then it's a bit more clear we're operating on the contact pointer.